test/hotspot/jtreg/compiler/loopopts/InvariantCodeMotionReassociateCmp.java

```diff
@@ -0,0 +1,229 @@
+/*
+ * Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
```

Comment:
Does the Amazon copyright header not have a year  associated with it?

Reply:
I guess not, I see other files without a year. Still, a bit strange.

Reply:
Yes. Our open-source board advise us to do so.